.travis.yml

```diff
@@ -37,8 +37,6 @@ matrix:
     - os: linux
       env: PROJECT_DIR=examples/cryptopp TOOLCHAIN=android-ndk-r15c-api-21-armeabi-v7a-neon-clang-libcxx
     - os: linux
-      env: PROJECT_DIR=examples/cryptopp TOOLCHAIN=analyze
```

Comment:
Comment them out instead of removing. It will be easier to enable them back.

Reply:
Done.